application/static/process.js

```diff
@@ -58,6 +58,42 @@ function replaceInCell(type, cell, modelId, replace=0){
 }
 
 
+
+function computeRelativeDates(modelDate){
+    var now = new Date();
+    difference = (now - modelDate) / 1000; // convert from ms to s
+
+    var relativeTime; 
+    var relativeUnit;
+
+    if (difference < 60) {
```

Comment:
Minor nitpick:

While this works and is easy to follow. What worries me in these kind of approach is hard to find copy-paste errors. It'd be my preference to isolate the logic and the data, in an approach like:

```js
let [divisor, unit] = [[3600*24*8, null], [3600*24*7, "weeks"], [3600*24, "days"], [3600, "hours"], [60, "minutes"], [1, "seconds"]].filter(a => difference / a[0] > 1.)[0];
if (unit) {
    return `${Math.floor(difference / divisor)} ${unit} ago`
} else {
    return modelDate.toLocaleString()
}
```

So keep pairs of divisors and units and return the first pair where the quotient is greater than zero.

I haven't tested this and it's also a matter of taste, so I will just merge this in. If you agree, you can follow up with a new PR or otherwise keep these things in mind for the next time.